src/controllers/users.js

```diff
@@ -29,33 +30,44 @@ export async function registerUser(req, res) {
       email: user.email,
       hash: hash,
     });
-    const token = signToken(newUser)
-    newUser.setDataValue('token', token)
-    await newUser.save()
-    const responseData = userResponse(newUser)
+    const token = signToken(newUser.get());
+    newUser.setDataValue("token", token);
```

Comment:
Lets rethink how we are using the JWT tokens. We should not store them in the db, instead we can set an expiry time and during this time we know that the user is logged in. 
Have a look at this for more info: https://stackoverflow.com/questions/42763146/does-it-make-sense-to-store-jwt-in-a-database

Reply:
Interesting.  I will change jwt strategy.

Reply:
I added an expiration to the token and wont be stored in the database